ui/components/component-library/modal-overlay/modal-overlay.tsx

```diff
@@ -18,8 +18,8 @@ export const ModalOverlay: React.FC<ModalOverlayProps> = ({
   <Box
     className={classnames('mm-modal-overlay', className)}
     backgroundColor={BackgroundColor.overlayDefault}
-    width={BLOCK_SIZES.FULL}
-    height={BLOCK_SIZES.FULL}
+    width={BlockSize.Full}
+    height={BlockSize.Full}
     onClick={onClick}
     aria-hidden="true"
     {...props}
```

Comment:
We are using the following convention which prevents the typescript linting issue
```suggestion
    {...(props as BoxProps<C>)}
```

Reply:
Thank you for your assistance. I have made the necessary changes in the new commit.